<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationDetailsEditor</class>
 <widget class="QFrame" name="AnimationDetailsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="fpsLabel">
     <property name="text">
      <string>Frames Per Second</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Animation Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="fpsEdit">
     <property name="suffix">
      <string> fps</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="wrapModeLabel">
     <property name="text">
      <string>Wrap Mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="wrapModeCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
